Products/CMFPlone/tests/robot/test_linkintegrity.robot

```diff
@@ -60,9 +60,9 @@ I add a link in rich text
     Fill text to tinymce editor    foo
     Mark text foo in tinymce editor
     Click    //button[@aria-label="Insert/edit link"]
-    Click    //div[contains(@class,"linkModal")]//div[contains(@class,"content-browser-selected-items-wrapper")]//button[contains(@class,"btn-primary")]
-    Click    //div[contains(@class, "content-browser-wrapper")]//div[contains(@class, "levelColumns")]/div[1]/div[contains(@class, "levelItems")]/div[3]
-    Click    //div[contains(@class, "content-browser-wrapper")]//div[contains(@class, "levelColumns")]/div[contains(@class, "preview")]/div[contains(@class, "levelToolbar")]/button
+    Click    //div[contains(@class,"linkModal")]//div[contains(@class,"content-browser-selected-items-wrapper")]//a[contains(@class,"btn-primary")]
```

Comment:
locally here, this is a button not an anchor
Click    //div[contains(@class,"linkModal")]//div[contains(@class,"content-browser-selected-items-wrapper")]//button[contains(@class,"btn-primary")]

Reply:
Did you checkout the staticresources branch mentioned above?  There it was changed to an Anchor because the button prevents the form submit when pressing ENTER